### PR TITLE
Change ppp iframe link,display error message in surge pages

### PR DESCRIPTION
**Pull request by utsukta1**
## Addresses:
- https://github.com/IFRCGo/go-web-app/issues/431
- https://github.com/IFRCGo/go-web-app/issues/429

## Changes
- On three-w/projects the PPP iframe link changed in Global Three W
- Add error message in Rapid response Personnel table

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by shreeyash07 — APPROVED** [review]

File: src/views/EmergencySurge/RapidResponsePerosnnelTable/index.tsx [hunk 1]
```diff
@@ -140,10 +140,15 @@ export default function RapidResponsePersonnelTable(props: Props) {
                 'country_from',
                 strings.personnelTableDeployedParty,
                 (item) => item.country_from?.society_name,
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (item.country_from?.id) {
```
Comment (shreeyash07) [inline src/views/EmergencySurge/RapidResponsePerosnnelTable/index.tsx]: use isDefined or isNotDefined from fujs

**Review by shreeyash07 — CHANGES_REQUESTED** [review]

File: src/views/AllDeployedPersonnel/index.tsx [hunk 2]
```diff
@@ -131,10 +132,17 @@ export function Component() {
...
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (isDefined(item.country_from?.record_type === 3)) {
+                        return {
+                            to: undefined,
+                        };
+                    }
```
Comment (samshara) [inline src/views/AllDeployedPersonnel/index.tsx:140]: Let's add a NOTE here.
// NOTE: we don't have the correct mapping for region

File: src/views/EmergencySurge/RapidResponsePersonnelTable/i18n.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
         "personnelTableType": "Type",
         "personnelTableDeployedTo": "Deploying To",
         "personnelTableDeployedParty": "Deploying Party",
-        "deployedPersonnelViewAll": "View All Deployed Personnel"
+        "deployedPersonnelViewAll": "View All Deployed Personnel",
+        "emergencySurgePartyNotFound": "sorry, this party has no one matching country to be shown"
```
Comment (samshara) [inline src/views/EmergencySurge/RapidResponsePersonnelTable/i18n.json:12]: Let's remove this

File: src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx [hunk 4]
```diff
@@ -140,10 +140,17 @@ export default function RapidResponsePersonnelTable(props: Props) {
...
                 (item) => item.country_from?.society_name,
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (isDefined(item.country_from?.record_type === 3)) {
+                        return {
+                            to: undefined,
+                        };
```
Comment (samshara) [inline src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx:147]: Let's add a note

Comment (samshara) [reply src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx:147]: Let's use a constant to denote region type.
For example 
export const TYPE_IMMINENT = 0 satisfies TypeOfDrefEnum;

**Pull request closed without merging**